Documentation/ApiOverview/Rte/RenderingInTheFrontend/Index.rst

```diff
@@ -0,0 +1,90 @@
+.. include:: /Includes.rst.txt
+.. index::
+   Rich text editor
+   RTE
+   see: RTE; Rich text editor
+   CKEditor
+.. _rte-rendering-frontend:
+
+=========================
+Rendering in the Frontend
+=========================
+
+The explanations on this page don't show how to display an RTE but rather, describe how
+rendering of content should be done in the frontend when it was entered with help
+of an RTE.
+
+.. note::
+
+   For including an RTE in the frontend you can read :ref:`Using a RTE in the frontend <rte-frontend-introduction>`.
+
+Fluid-templates
+===============
+
+Rendering in TYPO3 is nowadays done mostly with fluid-templates.
```

Comment:
Fluid templates (Fluid is always upper case, no dash between nouns or adjective and noun)